firedrake/slope_limiter/limiter.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABCMeta, abstractmethod
+
+
```

Comment:
So that you only expose the things you want in the "public" API, write:

```python
__all__ = ("Limiter", )
```

here.  Now, if someone does `from ...limiter import *` then they only get the names in the list in `__all__`.